CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## Unreleased
+### Additions and Improvements
+- Add support for transaction permissioning rules in Plugin API [#8365](https://github.com/hyperledger/besu/pull/8365)
```

Comment:
there's a Plugins heading now on line 29, can you move line 5 to that section, and you won't need line 4